slic-tools/middy-util.js

```diff
@@ -19,10 +17,15 @@ function middify (exports, options = {}) {
           log.debug({ request }, 'request')
         }
       }))
-      .use(httpEventNormalizer())
-      .use(httpJsonBodyParser())
-      .use(httpCors())
-      .use(httpErrorHandler())
+
+    if (options.isHttpHandler) {
```

Comment:
Latest version of the middy middlewares will throw if the event payload is not recognised as a http event so a new option `isHttpHandler` is introduced to conditionally add the related middlewares